test/unit/objects/account_test.py

```diff
@@ -278,3 +279,20 @@ def test_account_availability_api_get(self):
             self.assertEqual(availability.unavailable, [])
 
             self.assertEqual(m.call_url, account_availability_url)
+
+
+class ChildAccountTest(ClientBaseCase):
+    """
+    Test methods of the ChildAccount
+    """
+
+    def test_child_account_api_list(self):
```

Comment:
Could a test also be added for `LinodeClient.load(ChildAccount, ...)`?

Reply:
Of course! Just got it added